src/copilot/context/copilotHelper.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { commands, Uri } from "vscode";
+import { logger } from "../utils";
+
+export interface INodeImportClass {
+    uri: string;
+    className: string;  // Changed from 'class' to 'className' to match Java code
+}
+/**
+ * Helper class for Copilot integration to analyze Java project dependencies
+ */
+export namespace CopilotHelper {  
+    /**
+     * Resolves all local project types imported by the given file
+     * @param fileUri The URI of the Java file to analyze
+     * @returns Array of strings in format "type:fully.qualified.name" where type is class|interface|enum|annotation
+     */
+    export async function resolveLocalImports(fileUri: Uri): Promise<INodeImportClass[]> {
+        try {
```

Comment:
could you please check if the command exists before calling it?

Reply:
leverage `validateAndRecommendExtension` to do the check logic